service-commands/scripts/A

```diff
@@ -1,3 +1,8 @@
 #!/bin/bash
 cd $PROTOCOL_DIR/service-commands
-node scripts/setup.js $@
\ No newline at end of file
+
+if [[ "$1" == "create-instance" || "$1" == "setup" ]]; then
```

Comment:
i see the benefit of having this script so we can change the python script without having to move it to the bin directory. but can we abstract this if/else inside the python so ideally we never have to touch this script again? i feel like this is ripe for confusion for the next person adding something here